jerry-ext/arg/arg.c

```diff
@@ -16,7 +16,7 @@
 #include "arg-internal.h"
 #include "jerryscript-ext/arg.h"
 #include "jerryscript.h"
-
+#include "jerryscript-port.h"
```

Comment:
Is this still necessary? `jerryscript.h` includes `jerryscript-compiler.h`, so the macros are visible. Moreover, `jerryscript-port.h` should only be included when port functions are used (which is not the case here).

Same comment goes to module.c, too.

Reply:
include jerryscript-compiler.h instead? OK, give a try.

Reply:
No, there should be no need for any new include: jerryscript-compiler.h should already be included transitively.